_security-analytics/api-tools/mappings-api.md

```diff
@@ -109,10 +109,16 @@ POST /_plugins/_security_analytics/mappings
 ---
 ## Get Mappings
```

Comment:
@Naarcha-AWS This needs an introductory sentence before jumping into the next section. For example, "The Get Mappings API retrieves mapping definitions for an index or indexes."